client/lib/directly/index.js

```diff
@@ -110,6 +103,30 @@ export function initialize() {
 			}
 		} );
 	} );
+}
+
+/**
+ * Initializes the RTM widget if it hasn't already been initialized. This sets up global
+ * objects and DOM elements and requests the vendor script.
+ *
+ * @returns {Promise} Promise that resolves after initialization completes or fails
+ */
+export function initialize() {
+	if ( directlyPromise instanceof Promise ) {
+		return directlyPromise;
```

Comment:
I don't think this is the best way to prevent double initialization. What happens if an error occurs in the `getDirectlyConfiguration` network call and the user navigates away from the page and then returns? I think the way this is setup is that the page will call `directly.initialize()` again and it will return an old promise that has already been rejected. The user will never get to use Directly unless they do a page refresh to get a new `directlyPromise`.